Lib/fontTools/ttLib/__init__.py

```diff
@@ -15,3 +17,70 @@ def debugmsg(msg):
 
 from fontTools.ttLib.ttFont import *
 from fontTools.ttLib.ttCollection import TTCollection
+
+
+def main(args=None):
+	"""Open font with TTFont() or TTCollection()
+
+    ./fonttools ttLib [-oFILE] [-yNUMBER] files...
+
+	If multiple files are give on the command-line,
+	they are each opened (as a font or collection),
+	and added to the font list.
+
+	If -o (output-file) argument is given, the font
+	list is then saved to the output file, either as
+	a single font, if there is only one font, or as
+	a collection otherwise.
+
+	If -y (font-number) argument is given, only the
+	specified font from collections is opened.
+
+	The above allow extracting a single font from a
+	collection, or combining multiple fonts into a
+	collection.
+
+    If --lazy or --no-lazy are give, those are passed
+    to the TTFont() or TTCollection() constructors.
+	"""
+	from fontTools import configLogger
+
+	if args is None:
+		args = sys.argv[1:]
+
+	import getopt
+	options, files = getopt.getopt(args, "o:y:",
```

Comment:
i'd have expected to use argparse instead of getopt, but it's ok

Reply:
argparse is so much better. The Python docs for getopt start with this highlighted box:

> Note The [getopt](https://docs.python.org/3/library/getopt.html#module-getopt) module is a parser for command line options whose API is designed to be familiar to users of the C getopt() function. Users who are unfamiliar with the C getopt() function or who would like to write less code and get better help and error messages should consider using the [argparse](https://docs.python.org/3/library/argparse.html#module-argparse) module instead.

I think fonttools should use argparse for all new code.